packages/argo-admin/src/extension-api/container/index.ts

```diff
@@ -1,9 +1,12 @@
-import {ExtensionPoint, ExtensionApi} from '../extension-points';
+import {ExtensionPoint, ExtensionApi} from '../../extension-points';
```

Comment:
No docs for this?

Reply:
I'm still thinking about how to document extension points. They're not near as simple as component, or even APIs, for which listing the props + a brief intro pretty much covers.

"listing the props" for an extension point mean something more like a list of allowable components and APIs, plus the actual props for the data API. however, the way we've implemented extension points is quite complicated to parse with tsdoc. I'm treating it as a separate task, and thinking about how automated it should really be.

Reply:
tldr this is my next task after the other APIs

Reply:
same problem as `data` - the way we've typed the containers using generics on the extension point is quite hard to parse with tsdoc. there's like 3 levels of tracing generics to get from this file to

```
data: {
  productId: string;
  variantId?: string;
};
```

Reply:
we also didn't have an example for useContainer in the existing docs: https://github.com/Shopify/argo-admin-template/blob/master/docs/Utilities/Container.md

Reply:
Maybe we should remove the docs for `useData` to be consistent then?

Reply:
i'm gonna write an example, better to have than not.